### DISCUSSION TITLE
Fritz!Box 7490

### DISCUSSION THREAD

**Discussion by Fritzz2022** [PeterPawn/YourFritz#53] [Q&A]
Guten Tag. 

Meine Fritz!Box 7490 Blinkt Power/DSL und Internet nach dem einstecken an einer Steckdose. Festnetz und Wlan Blinken im 20 sekunden Takt.

Ich habe mich mit AVM auseinandergesetzt die mich zu der recover.exe datei weitergeleitet haben. 

Daraus wurde der Fehler gelesen das die Fritzbox nicht erreichbar ist.
AVM hat kurzen Prozess gemacht und mich einfach weiter geschickt zu meinem Anbieter, denn ich erhalte einen Austausch erst mit einem Kaufbeleg.

Dann bin ich weiter zu 1&1 gegangen und die meinten nach drei abgebrochenen Anrufen keinen Kaufbeleg ausstellen zu können. 

Ich stecke fest mit einer Fritzbox und versuche diese zu retten...

Können Sie mir dabei behilflich sein?

**Comment by PeterPawn** [comment]
> Festnetz und Wlan Blinken im 20 sekunden Takt.

Das gibt es normalerweise NICHT. Es hört sich für mich eher so an, als würde die gesamte Box alle 20 Sekunden neu starten.

Das ist auch kein neues/einmaliges Fehlerbild - je nachdem, welche Komponenten es da tatsächlich zerwürgt hat, kann sich - für beschränkte Einsatzfälle, wenn z.B. die analogen Telefonanschlüsse betroffen sein sollten und man aber gar keine Telefonie-Funktionen benutzen will - das Verwenden einer angepaßten Firmware lohnen (oder auch nicht).

Ich empfehle die Suche im IPPF (www.ip-phone-forum.de) nach Benutzern mit ähnlichen Problemen (üblicherweise sollte "bootloop 7490" ausreichen, dann ist eben die Ergebnismenge entsprechend gewaltig) und das "Abarbeiten" der Lösungsvorschläge, die den Fragenden dort geholfen haben (oder manchmal auch nicht).

Angesichts des "Restwerts" einer 7490 stellt sich (je nachdem, welche der vielen verschiedenen Funktionen dieser Box man wirklich nutzen will) irgendwann auch die Sinnfrage, ob sich weiterer Aufwand (jenseits des Aspekts, daß man dabei Erfahrungen gewinnen kann) noch lohnt.

> Daraus wurde der Fehler gelesen das die Fritzbox nicht erreichbar ist.

Wie genau wurde das denn "ausgelesen"? Welche Daten wurden AVM zur Verfügung gestellt? Irgendwie glaube ich da eher an ein "Mißverständnis" - denn eine "Schlußfolgerung", daß eine Box bei einem Recovery-Versuch nicht korrekt reagiert hat, deutet noch nicht zwingend auf ein Problem mit der Box hin (https://www.ip-phone-forum.de/threads/wie-recovere-ich-eigentlich-richtig.299790/), eine Reboot-Schleife mit einer Zyklusdauer von 20 Sekunden hingegen schon.

Aber praktisch alle (mir bekannten) Tricks und Kniffe, wie man so einen Fehler weiter eingrenzen kann, finden sich auch in entsprechenden Threads im IPPF - ich würde eher dort fortfahren (mit dem Suchen, Lesen, Verstehen). Eine "Garantie" für den Erfolg gibt es weder hier, noch im IPPF.

**Comment by Fritzz2022** [comment]
nach langer recherche habe ich nichts zu mienem fall gefunden. es wird über einen zugriff gesprochen, ein branding oder flashen. nur weiss ich nicht wie ich darauf zugreifen könnte. ggf könnten wir uns über discord zusammensetzten dass sie mir helfen?

**Comment by PeterPawn** [comment]
https://www.google.com/search?q=site%3Aip-phone-forum.de+7490+bootloop

In den dabei gefundenen Threads wird dann auch erklärt, wie eine "Fehlerbeschreibung" aussehen sollte, damit andere diese entsprechend nachvollziehen können und für viele - häufiger auftretende - Probleme sind da auch schon passende Lösungsvorschläge nachzulesen.

---

Und mal ganz ehrlich ... Discord? Sorry, das geht mal GAR NICHT, weil das u.a. unter dem Aspekt des Datenschutzes ein einziger Albtraum ist. Ich empfehle hier die (genaue) Lektüre von https://discord.com/privacy - speziell den Abschnitt "How we share your information". 

Wenn man so etwas schon vorschlagen möchte (was immer noch nicht heißt, daß ich von "Privatkonsultationen" begeistert wäre), dann sollten das wenigstens Dienste sein, die eine Ende-zu-Ende-Verschlüsselung bereithalten, damit da nicht jeder Honk "mithören" kann. Da sind dann Boards wie das IPPF die deutlich bessere Lösung, denn da ist von Beginn an klar (zumindest sollte das jedem stets gegenwärtig sein - und das gilt auch für die Plattform hier), daß es eben KEINE Punkt-zu-Punkt-Kommunikation ist.
